build/Targets/VSL.Versions.targets

```diff
@@ -72,13 +72,10 @@
     </BuildNumberPart2>
 
     <!-- Only set when building RTM with no dependencies on pre-release packages
```

Comment:
@tmat Agreed the number of variables and calls to BuildNuGet.csx can be reduced. Let's handle that in a subsequent PR.